packages/image/src/checkWebPSupport.ts

```diff
@@ -33,9 +32,8 @@ const features = Object.keys(kTestImages).filter((key): key is Feature =>
 
 /**
  * Checks if the browser supports all key WebP features.
- * @returns {Promise<boolean>} Returns a promise that resolves with a boolean indicating if all key WebP features are supported.
  */
-async function checkWebPSupport(): Promise<boolean> {
+async function checkWebPSupport() {
   const results = await Promise.all(features.map(checkWebPFeatureSupport));
   return results.every(Boolean);
 }
```

Comment:
How about this
1. I don't think always async await is good to read.
2. named export directly will make developer use name checkWebPSupport

```suggestion
export const checkWebPSupport = () => Promise.all(features.map(checkWebPFeatureSupport)).then((results) => results.every(Boolean))
```